### DISCUSSION TITLE
Add tags through a patchset.

### DISCUSSION THREAD

**Discussion by zaphinath** [crossplane/crossplane#3389] [General]
We want to patch all our resources by adding various tags through a patchset. Currently we are thinking about doing something like the following:

  - name: stackIdConfig
    patches:
      - type: FromCompositeFieldPath
        fromFieldPath: "spec.parameters.clusterName"
        toFieldPath: spec.forProvider.tags[StackId].value

The question is does this create a key "StackId" and then add the value or do we need to add the key first either in the patchset or resource definition? The docs are super clear and we haven't been able to find some good examples on appending to this array of objects.

Thanks.

**Comment by MisterMX** [comment]
(I believe this should be discussion and not an enhancement issue)

@zaphinath yes, patching into a field will create the full path. You don't have to create the parent fields before that.
